docs/resource/k8s.md

```diff
@@ -356,9 +356,9 @@ A CronJob creates Jobs on a repeating schedule.
 | [`k8s.cronjob.annotation.<key>`](/docs/attributes-registry/k8s.md) | string | The annotation key-value pairs placed on the CronJob. [1] | `4`; `` | `Opt-In` | ![Development](https://img.shields.io/badge/-development-blue) |
 | [`k8s.cronjob.label.<key>`](/docs/attributes-registry/k8s.md) | string | The label key-value pairs placed on the CronJob. [2] | `weekly`; `` | `Opt-In` | ![Development](https://img.shields.io/badge/-development-blue) |
 
-**[1] `k8s.cronjob.annotation`:** The `<key>` being the annotation name, the value being the annotation value, even if the value is empty.
+**[1] `k8s.cronjob.annotation.<key>`:** The `<key>` being the annotation name, the value being the annotation value, even if the value is empty.
```

Comment:
Since we are on it maybe to also fix the k8s.pod too? It seems everything is in the `brief` there: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md?plain=1#L150-L151

Reply:
could you elaborate? Here I'm just fixing jinja to show `<key>` in the template attribute notes, maybe you meant to comment on https://github.com/open-telemetry/semantic-conventions/pull/2175 ? I think I make all template briefs consistent there including k8.pod, but let me know if something is missing. Thanks!